src/styles/index.scss

```diff
@@ -1,3 +1,128 @@
-body {
-  margin: 0;
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white-accent: #fff;
+
+.product-card {
+  font-family: Roboto, Arial, Helvetica, 'Segoe UI', sans-serif, system-ui;
+  font-weight: 500;
+  font-size: 12px;
+  color: $main-accent;
+
+  &__page {
+    margin: 0;
+  }
+
+  &__all {
+    box-sizing: border-box;
+    width: 200px;
+    border: 1px solid #f3f3f3;
+    background: $white-accent;
+    border-radius: 5px;
+  }
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    background-image: url(../images/imac.jpeg);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: 160px 134px;
+    margin: 32px 19px 40px;
```

Comment:
The background image URL should be wrapped in quotes for consistency and to prevent potential issues with certain characters or spaces in the file path. Change it to `background-image: url('../images/imac.jpeg');`.